workitem/workitemtype.go

```diff
@@ -180,6 +180,7 @@ func (wit WorkItemType) ConvertWorkItemStorageToModel(workItem WorkItemStorage)
 		if err != nil {
 			return nil, errors.WithStack(err)
 		}
+		result.Fields[SystemOrder] = workItem.ExecutionOrder
```

Comment:
This looks like a hack. Are the work items that don't have the SystemOrder field set?